.github/workflows/build_test.yml

```diff
@@ -0,0 +1,110 @@
+name: Build and Test
+
+on:
+  pull_request:
+    branches: [master]
+
+jobs:
+  test:
+    runs-on: ${{ matrix.os }} # compiles and test on Ubuntu
+
+    strategy:
+      matrix:
+        os: [ubuntu-latest]
+        java: ["11"]
+      fail-fast: false
+
+    steps:
+    - name: Checkout PR branch
+      uses: actions/checkout@v2
+      with:
+        ref: ${{ github.event.pull_request.head.sha }}
```

Comment:
Will defeat baseline check

Reply:
What do you mean?

Reply:
See the discussion in jbosstools-base

Reply:
`fetch-depth: 0` ?

Reply:
and what of the discussion will help to pass baseline check?

Reply:
https://github.com/jbosstools/jbosstools-base/pull/695/commits/2c0748e97a63499869c5c8d7c92d46f66361c1e8

Reply:
thanks

Reply:
added.